test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/ViewEngineController.ViewWithPaths.txt

```diff
@@ -0,0 +1,21 @@
+﻿<Layout>
+    /Views/ViewWithPaths/_Layout.cshtml
+    /Views/ViewWithPaths/Index.cshtml
+</Layout>
+
+<ViewStart>
+    Views\ViewWithPaths\_ViewStart.cshtml
```

Comment:
> Interestingly, the VIewStart path generation is the only one that goes through Path::GetDirectoryName which normalizes the directory separator char. The other ones are produced by the view engine. Not super sure if there's a nice way to fix this or is it's particularly interesting to address this.

IMO worth considering in a follow-up. Will be wierd if half of the system normalizes paths and the other doesn't. Also recall our discussion last week about the hidden cost of `Path.GetDirectoryName`